theia-go-docker/next.package.json

```diff
@@ -1,22 +1,21 @@
 {
     "private": true,
     "dependencies": {
+        "@theia/plugin-ext-vscode": "next",
```

Comment:
for your consideration: @theia/preview would be useful to have, to render Markdown. I guess that would be true for all images

Reply:
added `@theia/preview`